### PR TITLE
update imodel content tree to imjs 2.15.2

### PR DESCRIPTION
**Pull request by jason-crow**
updating imodel-content-tree to imjs 2.15.2 in order to resolve a breaking change to the usePresentationTreeNodeLoader

### CODE REVIEW

**Review by aruniverse — APPROVED** [review]

**Review by diegopinate — APPROVED** [review]

**Review by calebmshafer — APPROVED** [review]

File: common/changes/@bentley/imodel-content-tree-react/update-imodel-content-tree-2.15.2_2021-05-13-16-01.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/imodel-content-tree-react",
+      "comment": "update to imjs 2.15.2 to resolve breaking change to usePresentationTreeNodeLoader",
+      "type": "patch"
```
[the remark this answers is not in the document]
Comment (jason-crow) [reply common/changes/@bentley/imodel-content-tree-react/update-imodel-content-tree-2.15.2_2021-05-13-16-01.json:6]: the instructions listed always setting the type to patch... but i did find that strange.

**Review by JoeZman — APPROVED** [review]

File: common/changes/@bentley/imodel-content-tree-react/update-imodel-content-tree-2.15.2_2021-05-13-16-01.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (jason-crow) [reply common/changes/@bentley/imodel-content-tree-react/update-imodel-content-tree-2.15.2_2021-05-13-16-01.json:6]: "Every change must be described with a change log: Run "rush change" on your committed and always choose "patch" as the change type. Commit your change log along with your pull request."

[the remark this answers is not in the document]
Comment (aruniverse) [reply common/changes/@bentley/imodel-content-tree-react/update-imodel-content-tree-2.15.2_2021-05-13-16-01.json:6]: Well, to be fair this pkg is 0.x, so its more acceptable for it to be breaking

[the remark this answers is not in the document]
Comment (calebmshafer) [reply common/changes/@bentley/imodel-content-tree-react/update-imodel-content-tree-2.15.2_2021-05-13-16-01.json:6]: @aruniverse true, maybe we bump the minor version then?

@jason-crow not sure why they say that :)

[the remark this answers is not in the document]
Comment (aruniverse) [reply common/changes/@bentley/imodel-content-tree-react/update-imodel-content-tree-2.15.2_2021-05-13-16-01.json:6]: If anything, bump the minor, idk about the major. It is how we do things in the itwin-viewer, but as we've seen, we quickly go through majors due to core

[the remark this answers is not in the document]
Comment (jason-crow) [reply common/changes/@bentley/imodel-content-tree-react/update-imodel-content-tree-2.15.2_2021-05-13-16-01.json]: set it to minor

**Pull request merged**